maintainers/maintainer-list.nix

```diff
@@ -5508,6 +5508,12 @@
       fingerprint = "9484 44FC E03B 05BA 5AB0  591E C37B 1C1D 44C7 86EE";
     }];
   };
+  dvn = {
```

Comment:
There is a preference for using your github handle here, so `dvn0`.

Reply:
Okay, I'll change it, though I don't really like that. It's not clear this is preferred in the linked documentation on adding maintainers, for the record. I know because I checked before making this commit.

Reply:
@dvn0 it's stated at the top of this file https://github.com/dvn0/nixpkgs/blob/master/maintainers/maintainer-list.nix#L29

Reply:
@marcusramberg  I already changed it. Also, I don't think it's reasonable to expect someone to search through comments to find additional instructions when there is an actual documentation page that people are supposed to be following... Not sure if that's what you mean to imply, but its the inferred implication.